lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const url = require('url');
+const Audit = require('../audit');
+const Formatter = require('../../formatters/formatter');
+
+class ExternalAnchorsUseRelNoopenerAudit extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Performance',
+      name: 'external-anchors-use-rel-noopener',
+      description: 'Site opens external anchors using rel="noopener".',
+      helpText: 'links that open in a new tab should use <code>target="_blank"</code> and <code>rel="noopener"</code> so the <a href="https://jakearchibald.com/2016/performance-benefits-of-rel-noopener">opening page\'s performance does not suffer.</a>',
+      requiredArtifacts: ['URL', 'AnchorsWithNoRelNoopener']
+    };
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    if (typeof artifacts.AnchorsWithNoRelNoopener === 'undefined' ||
+        artifacts.AnchorsWithNoRelNoopener === -1) {
+      return ExternalAnchorsUseRelNoopenerAudit.generateAuditResult({
+        rawValue: -1,
+        debugString: 'AnchorsWithNoRelNoopener gatherer did not run'
+      });
+    }
+
+    const pageHost = url.parse(artifacts.URL.finalUrl).host;
+    // Filter usages to exclude anchors that are same origin
+    const failingAnchors = artifacts.AnchorsWithNoRelNoopener.usages
+      .filter(anchorHref => url.parse(anchorHref).host !== pageHost)
+      .map(anchorHref => {
+        return {
+          url: anchorHref
```

Comment:
Did you want to add https://github.com/GoogleChrome/lighthouse/pull/912#discussion_r86694878?